gradio/event_queue.py

```diff
@@ -22,175 +22,146 @@ class Estimation(BaseModel):
 
 
 class Queue:
-    EVENT_QUEUE = []
-    EVENTS_PENDING_RECONNECTION = []
-    STOP = False
-    CURRENT = None
-    MAX_THREAD_COUNT = 1
-    DATA_GATHERING_STARTS_AT = 30
-    UPDATE_INTERVALS = 10
-    ACTIVE_JOBS: List[None | Event] = [None]
-    DELETE_LOCK = asyncio.Lock()
-    SERVER_PATH = None
-    DURATION_HISTORY_TOTAL = 0
-    DURATION_HISTORY_COUNT = 0
-    AVG_PROCESS_TIME = None
-    AVG_CONCURRENT_PROCESS_TIME = None
-    QUEUE_DURATION = 1
-    LIVE_UPDATES = True
-    SLEEP_WHEN_FREE = 0.05
-    MAX_SIZE = None
-    MAX_WAIT_PENDING_RECONNECTION = 120
-
-    @classmethod
-    def configure_queue(
-        cls,
+    def __init__(
+        self,
         live_updates: bool,
         concurrency_count: int,
         data_gathering_start: int,
         update_intervals: int,
         max_size: Optional[int],
     ):
-        """
-        See Blocks.queue() docstring for the explanation of parameters.
-        """
-        cls.LIVE_UPDATES = live_updates
-        cls.MAX_THREAD_COUNT = concurrency_count
-        cls.DATA_GATHERING_STARTS_AT = data_gathering_start
-        cls.UPDATE_INTERVALS = update_intervals
-        cls.ACTIVE_JOBS = [None] * cls.MAX_THREAD_COUNT
-        cls.MAX_SIZE = max_size
-
-    @classmethod
-    def set_url(cls, url: str):
-        cls.SERVER_PATH = url
-
-    @classmethod
-    async def init(
-        cls,
-    ) -> None:
-        run_coro_in_background(Queue.start_processing)
-        if not cls.LIVE_UPDATES:
-            run_coro_in_background(Queue.notify_clients)
-
-    @classmethod
-    def close(cls):
-        cls.STOP = True
-
-    @classmethod
-    def resume(cls):
-        cls.STOP = False
-
-    @classmethod
-    def get_active_worker_count(cls) -> int:
+        self.event_queue = []
+        self.events_pending_reconnection = []
+        self.stopped = False
+        self.max_thread_count = concurrency_count
```

Comment:
This used to be 1. Maybe you mean:
```suggestion
        self.max_thread_count = concurrency_count or 1
```